xnmt/search_strategy.py

```diff
@@ -32,8 +33,9 @@ def generate_output(self, decoder, attender, output_embedder, dec_state, src_len
 
       score += logsoftmax[cur_id]
       word_ids.append(cur_id)
+      attentions.append(attender.get_last_attention())
 
-    return word_ids, score
+    return {"word_ids": word_ids, "attentions": attentions}, score
```

Comment:
This is just a suggestion and could also be done later, but Python offers collections.namedtuple for a bit cleaner handling of these situations.

Reply:
Yeah I can do this. This shouldn't be too much problem.